x/vault/module/simulation.go

```diff
@@ -0,0 +1,60 @@
+package vault
+
+import (
+	"math/rand"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	simtypes "github.com/cosmos/cosmos-sdk/types/simulation"
+	"github.com/cosmos/cosmos-sdk/x/simulation"
+
+	"github.com/ardaglobal/ardaos/testutil/sample"
+	vaultsimulation "github.com/ardaglobal/ardaos/x/vault/simulation"
+	"github.com/ardaglobal/ardaos/x/vault/types"
+)
+
+// avoid unused import issue
+var (
+	_ = vaultsimulation.FindAccount
+	_ = rand.Rand{}
+	_ = sample.AccAddress
+	_ = sdk.AccAddress{}
+	_ = simulation.MsgEntryKind
+)
+
+//nolint:gocritic
+const (
+// this line is used by starport scaffolding # simapp/module/const
+)
+
+// GenerateGenesisState creates a randomized GenState of the module.
+func (AppModule) GenerateGenesisState(simState *module.SimulationState) {
```

Comment:
My understanding is the vault will create transactions with attestations that point to the objects in the storage layer as opposed to having it's own state on chain.

Reply:
this is just default module initiation. not sure what the final form of this module will be since building on your point, there vault module will most likely be an interface to the off chain storage